exercise_perseus_renderer/assets/src/vue/perseus.vue

```diff
@@ -214,6 +214,13 @@
           this.itemRendererFactory(this.itemRenderData, null),
           this.$refs.perseusContainer, () => { this.loading = false; }
         );
+
+        // Get all inputs in perseus and add event listeners to them.
+        // Here we dismiss answer error message on focus.
+        const perseusInputs = this.$refs.perseus.getElementsByTagName('input');
+        [].forEach.call(perseusInputs, (perseusInput) => {
```

Comment:
This would be better done as `Array.prototype.forEach.call`, rather then referencing an anonymous instance.